k8s/local/skaffold.local.full.yaml

```diff
@@ -0,0 +1,172 @@
+apiVersion: skaffold/v4beta6
```

Comment:
Great idea! I think this should be the default one to help potential new devs to get started.

The other one you can name `k8s/local/skaffold.local.light.yaml`.

Also: make sure to update the README!

Reply:
Why change the name to light?

Reply:
I think we should have these two:

`skaffold.local.yaml` <- default (full)
`skaffold.local.light.yaml` <- for a lighter restart. You can take another name, like "small", if you like that better.